.github/pester/release.tests.ps1

```diff
@@ -42,7 +42,7 @@ Describe "version.json file tests" {
     }
 
     It "version.json file releaseNotes property has been updated and URL last split on / does not match the latest git tag" {
-      $releaseNotesUrlSplitLast | Should -Not -Be $gitRepoLatestTag
+      $releaseNotesUrlSplitLast | Should -Be $gitRepoLatestTag
```

Comment:
I think the NOT here was valid as we don't want this property to be the same as the latest tag on the repo. Bear in mind this runs on PRs that have "release" in their branch name.